pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -320,6 +321,9 @@ func TestServiceEvents(t *testing.T) {
 	require.NoError(t, err)
 
 	done := make(chan struct{})
+	// TODO: the test is failing sometimes, need to review the test logic
+	//       and make sure that `once` solution is fine or fix this test according to logic
+	var once sync.Once
```

Comment:
@rolsonquadras Let me know if this is a solution, looks like we are closing channel more than once

```
 [aries-framework/did-exchange] 2019/10/28 11:36:06 UTC - didexchange.(*Operation).writeResponse -> ERROR Unable to send error response, failed to write
panic: close of closed channel

goroutine 6493 [running]:
github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange.TestServiceEvents.func1(0xb15a09, 0xb, 0xc0020a3560, 0x120, 0x120, 0x43d4c0, 0xc0026aae00)
	/home/soluchok/projects/aries-framework-go/pkg/restapi/operation/didexchange/didexchange_test.go:336 +0x20a
github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange.(*mockNotifier).Notify(0xc0000b9a80, 0xb15a09, 0xb, 0xc0020a3560, 0x120, 0x120, 0x0, 0x431deb)
	/home/soluchok/projects/aries-framework-go/pkg/restapi/operation/didexchange/didexchange_test.go:471 +0x80
github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange.(*Operation).sendConnectionNotification(0xc002346cd0, 0xc002684d50, 0x24, 0x7f7ca045a008, 0xc002341dc0)
	/home/soluchok/projects/aries-framework-go/pkg/restapi/operation/didexchange/didexchange.go:416 +0x5e4
github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange.(*Operation).handleMessageEvents(0xc002346cd0, 0xc002703f18, 0x2, 0x1)
	/home/soluchok/projects/aries-framework-go/pkg/restapi/operation/didexchange/didexchange.go:378 +0x182
github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange.(*Operation).startClientEventListener.func1(0xc002346cd0)
	/home/soluchok/projects/aries-framework-go/pkg/restapi/operation/didexchange/didexchange.go:362 +0x1fe
created by github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange.(*Operation).startClientEventListener
	/home/soluchok/projects/aries-framework-go/pkg/restapi/operation/didexchange/didexchange.go:356 +0x291
FAIL	github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange	1.907s
?   	github.com/hyperledger/aries-framework-go/pkg/restapi/operation/didexchange/models	[no test files]

```

Reply:
Once will be just a temp solution (looks fine). For some reason events are triggered twice, hence the execution goes inside if clause. There is a issue to investigate/fix - #607